### DISCUSSION TITLE
Decision Boundaries In Neural Network

### DISCUSSION THREAD

**Discussion by ChampPhil** [mrdbourke/pytorch-deep-learning#835] [Q&A]
I've finished Chapter 2 (Neural Network Classification) [ ends at 13:24:09] but I am struggling to understand the concept of non-linearity in a neural network. 

I understand the difference between a non-linear and linear function, but that aha! moment didn't occur, so I did some external research. Essentially, the goal of a classification problem is to find the optimal decision boundaries that lead to minimal loss, and a decision boundary can be thought of as a line/hyperplane that separates datapoints into classes.

However, what I don't understand is
        - What role does each neuron play in creating a local decision boundary/global decision boundary?
        - How can a neural network create decision boundaries, linear or nonlinear, at all. I thought a neural network was just a series of mathematical equations.
 
        - How are local decision boundaries (created by each neuron?) combined to make the global decision boundary?
